pyreferrer/referrer.py

```diff
@@ -45,7 +41,11 @@ class Types:
     def parse_query_string(url, parameters):
         if not parameters:
             return ''
-        query_params = parse_qs(url.query.encode('utf-8'), keep_blank_values=True)
+
+        url_query = url.query
+        if six.PY2:
```

Comment:
Nice! I wasn't aware of this compatibility library before but reading on it now it sounds great. Much clearer in the code to see what is going on too. 🙌